lib/libfst/meson.build

```diff
@@ -16,6 +16,8 @@ libfst = static_library(
     sources: libfst_sources,
     dependencies: libfst_dependencies,
     include_directories: config_inc,
+    install: true,
+    install_dir: install_libs_rpath,
```

Comment:
Is this required? We are statically linking against all libraries except `libgtkwave`.